src/components/BookDetails/BookDetails.jsx

```diff
@@ -0,0 +1,28 @@
+import Authors from "../BookComponents/Authors/Authors";
+import Rating from "../BookComponents/Rating/Rating";
+import Thumbnail from "../BookComponents/Thumbnail/Thumbnail";
+import styles from "./BookDetails.module.scss";
+
+const BookDetails = ({ onClick, book }) => {
+  const {
+    volumeInfo: {
+      authors,
+      averageRating,
+      imageLinks: { thumbnail = "" } = {},
+      title,
```

Comment:
Less complicated destructuring required if you use a function to clean the data the way you want.